integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/AuxiliaryImageUtils.java

```diff
@@ -0,0 +1,324 @@
+    // create auxiliary image using imagetool command if does not exists
+    LoggingFacade logger = getLogger();
+    if (! dockerImageExists(imageName, MII_BASIC_IMAGE_TAG)) {
+      logger.info("creating auxiliary image {0}:{1} using imagetool.sh ", imageName, MII_BASIC_IMAGE_TAG);
+      testUntil(
+          withStandardRetryPolicy,
+          createAuxiliaryImage(witParams),
+          logger,
+          "createAuxImage to be successful");
+    } else {
+      logger.info("!!!! auxiliary image {0}:{1} exists !!!!", imageName, MII_BASIC_IMAGE_TAG);
+    }
+
+    // push image to repo for multi node cluster
+    if (!DOMAIN_IMAGES_REPO.isEmpty()) {
+      logger.info("docker push image {0} to registry {1}", imageName, DOMAIN_IMAGES_REPO);
+      dockerLoginAndPushImageToRegistry(imageName + ":" + MII_BASIC_IMAGE_TAG);
+    }
+  }
+
+  /**
+   * Setup common env with wit params.
+   *
+   * @param witParams wit params
+   *
+   */
+  public static WitParams setupCommonWitParameters(WitParams witParams) {
+    // Set additional environment variables for WIT
+    checkDirectory(WIT_BUILD_DIR);
+    Map<String, String> env = new HashMap<>();
+    env.put("WLSIMG_BLDDIR", WIT_BUILD_DIR);
+
+    // For k8s 1.16 support and as of May 6, 2020, we presently need a different JDK for these
+    // tests and for image tool. This is expected to no longer be necessary once JDK 11.0.8 or
+    // the next JDK 14 versions are released.
+    String witJavaHome = System.getenv("WIT_JAVA_HOME");
+    if (witJavaHome != null) {
+      env.put("JAVA_HOME", witJavaHome);
+    }
+
+    String witTarget = ((OKD) ? "OpenShift" : "Default");
+
+    return witParams.target(witTarget).env(env).redirect(true);
+  }
+
+  /**
+   * Create a AuxImage.
+   *
+   * @param auxImage auxImage name
+   * @param modelList model list
+   * @param archiveList archive list
+   *
+   */
+  public static Callable<Boolean> createAuxiliaryImage(String auxImage, List<String> modelList,
+                                                       List<String> archiveList) {
+
+    return (() -> {
+      WitParams witParams =
+          new WitParams()
+              .modelImageName(auxImage)
+              .modelImageTag(MII_BASIC_IMAGE_TAG)
+              .modelFiles(modelList)
+              .modelArchiveFiles(archiveList);
+
+      return createAuxImageUsingWIT(witParams);
+    });
+  }
+
+  /**
+   * Create a AuxImage using WIT.
+   *
+   * @param witParams wit params
+   *
+   */
+  public static Callable<Boolean> createAuxiliaryImage(WitParams witParams) {
+
+    return (() -> createAuxImageUsingWIT(witParams));
+  }
+
+  /**
+   * Check Configured JMS Resource.
+   *
+   * @param domainNamespace domain namespace
+   * @param adminServerPodName  admin server pod name
+   * @param adminSvcExtHost admin server external host
+   */
+  public static void checkConfiguredJMSresouce(String domainNamespace, String adminServerPodName,
```

Comment:
This utility method should not be part of AuxiliaryImageUtils

Reply:
fixed